CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.3.6
```

Comment:
i think we should bump minor version. 1.4.0

Reply:
updated